pkg/apis/core/validation/validation.go

```diff
@@ -5311,6 +5316,11 @@ func validateWindowsSecurityContextOptions(windowsOptions *core.WindowsSecurityC
 		}
 	}
 
+	if len(windowsOptions.RunAsUserName) > maxRunAsUserNameLength {
+		errMsg := fmt.Sprintf("runAsUserName's length must be under %d characters", maxRunAsUserNameLength)
```

Comment:
"under or equal" as per the above check.
[1]

Reply:
I've set >= above instead. The KEP mentions a 256 character limit, which sounds to me that they should be "under".